app/models/patient.rb

```diff
@@ -122,12 +122,26 @@ class Patient < ApplicationRecord
 
   scope :archived,
         ->(team:) do
-          joins_archive_reasons(team:).where("archive_reasons.id IS NOT NULL")
```

Comment:
I think you can remove the `joins_archive_reasons` scope as well now as these were the only places it was being used.